### ISSUE TITLE
Service down when the website ID is incorrect

### ISSUE THREAD

**Issue by JerrySukin** [umami-software/umami#1970]
When I build a request with a website ID that doesn't exist, the service crashes and the nginx returns HTTP 502 error.

PM2 logs:
```
0|umami    | Error: Website not found: 3e96d432-e43d-40fa-8ddf-48ff2a892331
0|umami    |     at findSession (/data/umami/.next/server/chunks/3972.js:250:15)
0|umami    |     at async /data/umami/.next/server/chunks/3972.js:162:21
0|umami    | error Command failed with exit code 1.
0|umami    | /data/umami/.next/server/chunks/3972.js:250
0|umami    |         throw new Error(`Website not found: ${websiteId}`);
0|umami    |               ^

```

Enviroment:
umami v2.1.0
CentOS 7
Node.js v16.20.0
MySQL 5.7

**Comment by vFire** [comment]
same thing

**Comment by mikecao** [comment]
Fixed in v2.2.0.
